e2e/tests/ui-driven/src/create-flow-with-geospatial.spec.ts

```diff
@@ -99,12 +118,41 @@ test.describe("Flow creation, publish and preview", () => {
     await page.goto(
       `/${context.team.slug}/${serviceProps.slug}/published?analytics=false`,
     );
+
     await expect(
       page.locator("h1", { hasText: "Find the property" }),
     ).toBeVisible();
     await answerFindProperty(page);
     await clickContinue({ page });
 
+    await expect(
+      page.getByRole("heading", { name: "About the property" }),
+    ).toBeVisible();
+
+    // Check map component has geoJson content
+    await checkGeoJsonContent(page, mockMapGeoJson);
+
+    // Check property info is being shown
+    await expect(page.getByText("Test Street, Testville")).toBeVisible();
+    await expect(page.getByText("Residential - Semi Detached")).toBeVisible();
+    const changeButton = page.getByRole("button", {
+      name: "Change your Property type",
+    });
+
+    await changeButton.click();
+
+    // change button navigates the user back thro the service so we need to continue twice
+    await answerQuestion({
+      page: page,
+      title: "Assign property type",
+      answer: "Commercial",
+    });
+
+    await clickContinue({ page });
+    await expect(page.getByText("Commercial")).toBeVisible();
```

Comment:
nit: this would be slightly more clear to me if there were more verbose `expect` statements (or just more code comments) to confirm we've navigated _forwards_ to About the property _again_ before checking that the property type now displays our new selection of "Commercial" - eg
```
// "Continue" has navigated us back to PropertyInformation which should display updated selection
await expect(page.getByRole("heading", { name: "About the property" })).toBeVisible();
await expect(page.queryByText("Residential - Semi Detached")).toBeUndefined();
await expect(page.getByText("Commercial")).toBeVisible();
 ```